docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,216 @@
+
+![org_browser_small](../../../images/org_browser_small.jpg)
+
+
+Click Org Browser and scroll down to **Custom Objects**. The first object you see is Account. You can view the metadata of the **Account** object (and do a few more cool things, but we’ll get to those in a bit). 
+
+## Use SOQL Builder to Build and Run your First Query
+
+Code Builder comes preloaded with everything you need, to build and run a SOQL query as a part of the Salesforce extension pack. 
+
+Let’s build and run a simple SOQL query on the Account object:
+
+1. Press Ctrl+Shift+P (⇧⌘P)  to view the command palette.
+2. Type “SOQL” in the search box and then select the **SFDX: Create Query in SOQL Builder** command.
+3. Select  the following:
+
+    a. **Account** Object in **From** field.
+
+    b. **BillingState** and **NumberofEmployees** in **Fields**.
+
+    c. Press **Run Query** to run this simple SOQL Query on your org.
+
+4. You’re prompted to save your changes if you close the untitled file. 
+5. Save the changes to a file using a “.soql” extension.
+6. Let’s rerun the saved query – Right click the file and select **Open With…** then select SOQL Builder.
+The file opens in SOQL Builder and you can rerun or edit the query as you wish.
+
+**More Information**: For more information about building complex queries and much more see the  [SOQL Builder](https://developer.salesforce.com/tools/vscode/en/soql/soql-builder) in the Salesforce Extensions for Visual Studio Code documentation.
+
+## Connect to a Different Org
+During the course of development, you'll use different types of orgs for different stages. For example, it's common to use a Developer sandbox or Development Edition org during the development phase, and move to other sandbox types for integration, testing, and staging. Eventually, you'll deploy your changes to a production org. You can connect Code Builder to any of these orgs to deploy or retrieve metadata.
+
+To log into an org:
+
+1. From the command palette run **SFDX: Authorize an Org**.
+2. Select either sandbox for the login URL (`test.salesforce.com`) or a developer org if you have access to one.
+3. Enter an alias for the org, for example, dev_pro_sandbox or my_playground.
+4. Log in with the relevant username and password.
+5. You’re now connected to your org, and its name is visible in the status bar at the bottom.
+6. Once you authorize an org, we create a connected app that takes care of future authorizations so you don't have to continually log in. Just click the org’s name and then choose the previous org from the list.	
+
+## Create, Retrieve, and Deploy a Custom Field
+Let’s add a custom field to an object in our org and pull its metadata into our Code Builder project.
+
+First let’s add a custom field  – 
+
+1. From **Setup**, go to **Object Manager | Account**.
```

Comment:
This doesn't follow our standard for setup, although it doesn't offend me. You follow the standard "above" - whoops, I used a bad word. Heh.